queue_services/entity-emailer/src/entity_emailer/email_templates/common/whitespace-16px.html

```diff
@@ -1,2 +1,3 @@
-  <!-- empty paragraph for spacing -->
+  <!-- paragraph for spacing -->
+  <!-- height = 16 + 16 = 32px -->
   <p style="margin-top: 16px !important;">&nbsp;</p>
```

Comment:
So this template part is NOT 16px tall. It's actually 32px tall.

If you want me to rename this file (and update all the places it's imported), let me know.